### PR TITLE
Upload Flyer Multipart Form Data

### PR DESCRIPTION
**Pull request by vinnie4k**
## Overview

Rewrote the upload flyer backend call to use the new HTTP form data request instead of GraphQL mutations.

## Changes Made

- Installed Alamofire **(pronounced ”AH LUH MO FIRE”).**
- Updated our secrets to include the new endpoint.
- Used Alamofire to create a POST request that takes in multipart form data instead of a GraphQL mutation.
- Fixed a potential issue where users were able to authenticate twice by disabling the button until the network request is complete.
- Changed the error message from “Please try again later.” to “Please try again with a smaller image size.” to make it more clear.
    - Most of the time, the issue is with the image size so it’s fine to have a generic message like so.
- Updated version to 1.3.9 and build 30.

## Test Coverage

Tested with various image sizes from 100kb to 24mb. Everything seems to work fine and images that are too large will have the proper error message.

## Next Steps

Implement the organization flyer view page as well as editing and deleting flyer.

## Screenshots

These images are around 2mb which should be more than enough.

### CODE REVIEW

File: Volume/View Models/FlyerUploadViewModel.swift [hunk 1]
```diff
@@ -66,32 +67,43 @@ extension FlyerUploadView {
...
-                    if case let .failure(error) = completion {
-                        print("Error: CreateFlyerMutation failed on FlyerUploadView: \(error)")
+                .validate()
+                .response { [weak self] response in
+                    switch response.result {
+                    case .success:
+                        self?.uploadSuccessful = true
+                        self?.showSpinner = false
+                    case .failure(let error):
+                        print("Error: CreateFlyerRequest failed on FlyerUploadView: \(error)")
```
Comment (jjennifergu) [inline Volume/View Models/FlyerUploadViewModel.swift:102]: can this also be logged somewhere?

File: Volume/Views/Orgs Login/FlyerUploadView.swift [hunk 2]
```diff
@@ -388,7 +388,7 @@ struct FlyerUploadView: View {
                     Text(success ? "Flyer Published!" : "Oh no, something went wrong.")
                         .font(.newYorkMedium(size: 20))
 
-                    Text(success ? "Thank you for using Volume." : "Please try again later.")
+                    Text(success ? "Thank you for using Volume." : "Please try again with a smaller image size.")
```
Comment (jjennifergu) [inline Volume/Views/Orgs Login/FlyerUploadView.swift:391]:
```suggestion
                    Text(success ? "Thank you for using Volume." : "Please reduce the image file size and try again.")
```

**Review by jjennifergu — CHANGES_REQUESTED** [review]

File: Volume/View Models/FlyerUploadViewModel.swift (hunk 1 above)
Comment (vinnie4k) [reply Volume/View Models/FlyerUploadViewModel.swift:102]: Huh

Comment (jjennifergu) [reply Volume/View Models/FlyerUploadViewModel.swift:102]: is the error logged on backend

Comment (vinnie4k) [reply Volume/View Models/FlyerUploadViewModel.swift:102]: yea i remember doing it unless me or someone erased my code

Comment (jjennifergu) [reply Volume/View Models/FlyerUploadViewModel.swift:102]: approved

**Review by jjennifergu — APPROVED** [review]

**Review by tiffany-pann — APPROVED** [review]
Cool stuff! Is image uploading the only place you're using AF?

**Comment by vinnie4k** [comment]
> Cool stuff! Is image uploading the only place you're using AF?

Yea

**Pull request merged**